eng/pipelines/coreclr/perf.yml

```diff
@@ -80,199 +80,238 @@ jobs:
       buildConfig: release
       platforms:
       - Linux_x64
-      - windows_x64
-      - windows_x86
-      - Linux_arm64
-      - Linux_musl_x64
-      - windows_arm64
+      #- windows_x64
+      #- windows_x86
+      #- Linux_arm64
```

Comment:
I assume you are just testing and will enable Linux_arm64 for mono aot-llvm lanes too

Reply:
Yes, I will add AOT for Arm64 as well, but didn't want to use up the queue during testing.